libvast/src/system/disk_monitor.cpp

```diff
@@ -0,0 +1,132 @@
+#include "vast/system/disk_monitor.hpp"
+
+#include "vast/concept/parseable/from_string.hpp"
+#include "vast/concept/parseable/vast/uuid.hpp"
+#include "vast/directory.hpp"
+#include "vast/error.hpp"
+#include "vast/fwd.hpp"
+#include "vast/logger.hpp"
+#include "vast/path.hpp"
+#include "vast/system/archive.hpp"
+#include "vast/uuid.hpp"
+
+#include <caf/typed_event_based_actor.hpp>
+
+#include "caf/detail/scope_guard.hpp"
```

Comment:
```suggestion
#include <caf/detail/scope_guard.hpp>
```

Again, wrong header grouping. Please run `git format-branch`.

Reply:
`git format-branch` doesnt catch this.

Reply:
It catches the wrong grouping if and only if you are using angle brackets.